src/concrete/ml/sklearn/base.py

```diff
@@ -96,7 +102,7 @@
 # Enable rounding feature for all tree-based models by default
 # Note: This setting is fixed and cannot be altered by users
 # However, for internal testing purposes, we retain the capability to disable this feature
-os.environ["TREES_USE_ROUNDING"] = "1"
+os.environ["TREES_USE_ROUNDING"] = os.environ.get("TREES_USE_ROUNDING", "1")
```

Comment:
This env variable is for ROUNDING not for the sum of the outputs of the tree ensembles in fhe

Reply:
Still do we want to keep this env var?